be/src/exec/pipeline/sink/table_function_table_sink_operator.cpp

```diff
@@ -0,0 +1,237 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "table_function_table_sink_operator.h"
+
+#include <utility>
+
+#include "formats/parquet/file_writer.h"
+#include "glog/logging.h"
+
+namespace starrocks::pipeline {
+
+// TODO(letian-jiang): optimize
+StatusOr<std::string> column_to_string(const TypeDescriptor& type_desc, const ColumnPtr& column) {
```

Comment:
you need to test all types.

Reply:
How about reusing existing type conversion code such as `Column::debug_item`, `cast_expr.cpp` ?

Reply:
For some complex type such as `datetime`, cast to string is not trivial

Reply:
> For some complex type such as `datetime`, cast to string is not trivial

what do you mean for being non-trivial?

Reply:
> How about reusing existing type conversion code such as `Column::debug_item`, `cast_expr.cpp` ?

The naming should conform to open-sourced table formats. 
Therefore, it is better to have a dedicated function to acheive this.